lib/LanguagePicker.js

```diff
@@ -2,6 +2,7 @@ const ls = require('language-scripts');
 const dataOverrides = require('./overrides.json');
 
 function LanguagePicker(lang = 'en', config = {}) {
+  // FIXME: fallbackMap is not used!
```

Comment:
Look at line #35; fallbackMap maps in the incoming config.languageMap, then gets the relevant fallbacks into the general fallback list after the language requested.